### PR TITLE
data-science topic added

### PR DESCRIPTION
**Pull request by avinal**
### Thank you for contributing! Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---------------------------------------------------------------------

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288)
- [ ] Content (and my changes are in `index.md`)

> Please replace this line with an explanation of why you think these changes should be made.

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> Data-science is a highly active field with more than 15000 repositories/projects on GitHub. A significant part of this is data science tools. Adding this topic will help a contributor find and contribute easily.

### Something that does not neatly fit into the binary options above

- [ ] My suggested edits are not about an existing topic or collection, or at least not a single one
- [ ] My suggested edits are not about curating a new topic or collection, or at least not a single one
- [ ] My suggested edits conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> Since this is a topic there is no suitable image for this topic.

---------------------------------------------------------------------

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

File: topics/data-science/index.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+aliases: datasciences, data-science-project, data-science-algorithm
+display_name: Data Science
+short_description: Data science is an interdisciplinary field focused on extracting knowledge from typically large data sets.
+topic: data-science
+related: data-analysis, data-mining, machine-learning, big-data, data-visualization
+wikipedia_url: https://en.wikipedia.org/wiki/Data_science
+---
+Data science is an inter-disciplinary field that uses scientific methods, processes, algorithms, and systems to extract knowledge and insights from many structural and unstructured data. The field encompasses analysis, preparing data for analysis, and presenting findings to inform high-level decisions in an organization.
```
Comment (deniseyu) [inline topics/data-science/index.md:9]: Could we restructure this to be a little easier to read:
```suggestion
Data science is an inter-disciplinary field that uses scientific methods, processes, algorithms, and systems to extract knowledge from structured and unstructured data. Data scientists perform data analysis and preparation, and their findings inform high-level decisions in many organizations.
```

**Review by deniseyu — APPROVED** [review]
Thanks for the contribution, @avinal! I've made some suggestions for rewording the English, but am open to your feedback.

File: topics/data-science/index.md (hunk 1 above)
Comment (avinal) [reply topics/data-science/index.md:9]: I think rewording makes it easier to read for sure. It was very technical before. This is nice 😁

**Comment by avinal** [comment]
I think its really better after rewording 👍

**Pull request merged**